src/shared-components/avatar/style.ts

```diff
@@ -17,9 +15,11 @@ const determineSize = ({
   width: ${SIZES[avatarSize]}px;
 `;
 
-const AvatarImage = styled.img`
+const AvatarImage = styled.img<{
+  avatarSize: 'small' | 'medium' | 'large';
```

Comment:
As an FYI as we convert things over, since I hadn't added types to this (which was never flagged since `${determineSize}` pulled in the props implicitly), I saw a `ThemeType`-related error. Once I added this typing for `avatarSize` the compiler was able to interpret the theme typing correctly.